src/cocotb/share/lib/verilator/verilator.cpp

```diff
@@ -113,32 +114,22 @@ int main(int argc, char** argv) {
 #endif
 
     while (!Verilated::gotFinish()) {
-        // Call registered timed callbacks (e.g. clock timer)
-        // These are called at the beginning of the time step
-        // before the iterative regions (IEEE 1800-2012 4.4.1)
-        VerilatedVpi::callTimedCbs();
-
-        // Call Value Change callbacks triggered by Timer callbacks
-        // These can modify signal values
-        settle_value_callbacks();
-
-        // We must evaluate whole design until we process all 'events'
-        bool again = true;
-        while (again) {
-            // Evaluate design
-            top->eval_step();
-
-            // Call Value Change callbacks triggered by eval()
-            // These can modify signal values
-            again = settle_value_callbacks();
+        do {
+            // We must evaluate whole design until we process all 'events' for
```

Comment:
```suggestion
            // We must evaluate the whole design until we process all 'events' for
```

Reply:
Sorry, the rebase was weird and included already merged commits.